generators/server/templates/src/test/java/package/gateway/responserewriting/SwaggerBasePathRewritingFilterTest.java.ejs

```diff
@@ -80,13 +80,13 @@ class SwaggerBasePathRewritingFilterTest {
         context.setResponseGZipped(false);
         context.setResponse(response);
 
-        InputStream in = IOUtils.toInputStream("{\"basePath\":\"/\"}", StandardCharsets.UTF_8);
+        InputStream in = IOUtils.toInputStream("{\"servers\":[{\"url\":\"http://localhost:8080/services\"}]}", StandardCharsets.UTF_8);
```

Comment:
Line 75 should be instead:

```java
MockHttpServletRequest request = new MockHttpServletRequest("GET", "/services/service1/v3/api-docs");
```

Then, this line should be:

```java
InputStream in = IOUtils.toInputStream("{\"servers\":[{\"url\":\"http://localhost:8080\"}]}", StandardCharsets.UTF_8);

Reply:
ok